TKGame/Components/Concrete/Item_PhysicsComponent.cs

```diff
@@ -13,14 +13,23 @@
 
 namespace TKGame.Components.Concrete
 {
-    internal class Item_PhysicsComponent : PhysicsComponent
+    class Item_PhysicsComponent : PhysicsComponent
     {
         void PhysicsComponent.Update(Entity entity, GameTime gameTime/*, World &world*/)
         {
             Player player = Player.Instance;
 
-            // if player is within pickup range of item
-            // remove item from map and add item to player's inventory
+            if (player != null)
+            {
+                Vector2 playerPosition = player.Position;
+                // if player is within pickup range
+                if (playerPosition.X == entity.Position.X &&
+                    playerPosition.Y >= entity.Position.Y - 12 &&
+                    playerPosition.Y <= entity.Position.Y + 12)
```

Comment:
It would be better to give the Item a HitBox (CollideComponent) and use the HitBox.Instersects() function. That way you don't need to do any pixel measurements or hard code values. Plus the conditional will look nicer